scss/input/_layout.scss

```diff
@@ -191,6 +199,122 @@
         }
     }
 
+    // Forms
+    $form-spacer: $padding-x * 2 !default;
+
+    .k-form,
+    .k-form-inline {
+        font-size: $font-size;
+        line-height: $line-height;
+        padding: $form-spacer;
+
+        fieldset {
+            border-width: 1px 0 0;
+            border-style: solid;
+            margin: ($form-spacer * 2) 0;
+            padding: 0;
+
+            &:first-child:first-of-type {
+                margin-top: 0;
+            }
+
+            &:last-child:last-of-type {
+                margin-bottom: 0;
+            }
+
+            legend {
+                font-size: $font-size-sm;
+                font-weight: 600;
+                line-height: 1;
+                margin-bottom: $padding-y-lg * 2;
+                text-transform: uppercase;
+                padding: 0 $padding-x 0 0;
+                width: auto;
+            }
+        }
+
+        .k-form-field {
```

Comment:
This class is specific to some sections in a form, so I prefer keeping the selector as it is